Frontend/ui-common/src/Components/TipsView/View.purs

```diff
@@ -209,3 +213,18 @@ iOSTipsView push state =
   where
   tipsMargin :: Int -> Int -> Array String -> Margin
   tipsMargin index listIndex listItem = Margin 0 (if listIndex > 0 then 8 else 0) (if (index + 1) == (length listItem) then 0 else 8) 0
+
+getPreviousVersion :: String -> String 
```

Comment:
@PraveenGongada Why is this version check required ?

Reply:
If its required can we reuse existing function

Reply:
This check I added for flexBox when doing chat, for SDK's this will be useful so added it here, this function need to be release level so added again.

Reply:
Can we reuse the function ?